diff --git a/setup_spindle_environment b/setup_spindle_environment
index 8043a54..2bff058 100755
--- a/setup_spindle_environment
+++ b/setup_spindle_environment
@@ -91,6 +91,7 @@ sed -i $ETC_SCHROOT_SPINDLE/nssdatabases -e "/^shadow$/d"
 sed -i $ETC_SCHROOT_SPINDLE/nssdatabases -e "/^group$/d"

 debootstrap \
+  --no-check-gpg \
   --include="qemu,bash-completion,augeas-tools,debootstrap,less,\
   sudo,parted,openssh-client,e2fsprogs,dosfstools,squashfs-tools,bzip2,git,zerofree" \
   wheezy "$TARGET_DIR" http://http.debian.net/debian || die "Debootstrap failed"
